Jenkinsfile

```
@@ -43,15 +45,11 @@ pipeline {
         script {
             if (env.BRANCH_NAME == 'master')
                 stage('notify master branch fail') {
-                    withCredentials([string(credentialsId: 'slack-token', variable: 'TOKEN')]) {
-                        sh '''
-                        echo '{"text":"Attention! assisted-installer master branch subsystem test failed, see: ' > data.txt
-                        echo ${BUILD_URL} >> data.txt
-                        echo '"}' >> data.txt
-                        curl -X POST -H 'Content-type: application/json' --data-binary "@data.txt"  https://hooks.slack.com/services/$TOKEN
-                        '''
-
+                    script {
+                        def data = [text: "Attention! assisted-installer branch  test failed, see: ${BUILD_URL}"]
```

Missing branch name and you have 2 spaces.
```suggestion
                        def data = [text: "Attention! assisted-installer master branch build failed, see: ${BUILD_URL}"]
```

---

Please use a generic solution
```
"Attention! ${BUILD_TAG} job failed, see: ${BUILD_URL}"
```